components/squad/squad_custom.lua

```diff
@@ -0,0 +1,38 @@
+local Squad = require('Module:VoganRL/Squad')
+local SquadRow = require('Module:VoganRL/Squad/Row')
+local Json = require('Module:Json')
+
+
+local CustomSquad = {}
+
+function CustomSquad.run(frame)
+	local squad = Squad()
+	squad:init(frame):title():header()
+
+	local args = squad.args
+
+	local index = 1
+	while args['p' .. index] ~= nil do
+		local player = Json.parseIfString(args['p' .. index])
+		local row = SquadRow(frame, player.role)
+		row:id({
+			player.id,
+			flag = player.flag,
+			link = player.link,
+			captain = player.captain,
+			role = player.role,
+		})
```

Comment:
Loan Icon support missing (comment not specific the to the place of the comment)

Reply:
Don't know what you're referring to. Can you help me?

Reply:
Badly explained by me. The Team Icon indicating a player's "real" team whilst they're on loan.
![image](https://user-images.githubusercontent.com/3426850/140377114-3801e1ed-c575-457e-83e3-5a8a9adc9645.png)
https://liquipedia.net/rainbowsix/Guidance_Gaming

Reply:
![image](https://user-images.githubusercontent.com/5138348/140491312-c5a9853e-1058-432f-aa0b-e526794c0c44.png)